src/2-pages/Budgets/EnvelopeEditDialog/EnvelopeEditDialog.tsx

```diff
@@ -69,6 +69,7 @@ export const EnvelopeEditDialog: FC = () => {
       group: envelope?.group || '',
       comment: envelope?.comment || '',
       currency: envelope?.currency || defaultCurrency,
+      expendByDefault: envelope?.expendByDefault || true,
```

Comment:
this value will be always true
use 
```suggestion
      expendByDefault: envelope?.expendByDefault ?? true,
```